src/frontend/komponenter/Behandling/Blankett/Blankett.tsx

```diff
@@ -51,14 +54,34 @@ const Blankett: React.FC<Props> = ({ behandlingId }) => {
         });
     };
 
+    const erBehandlingÅpen = (status: BehandlingStatus): boolean => {
+        if (
```

Comment:
Dette uttrykket er det samme som `status === true` - som aldri vil slå til. 
Du kan gjøre noe slik `[BehandlingStatus.UTREDES, BehandlingStatus.OPPRETTET].includes(status)` for å sjekke flere verdier av status.

Reply:
Det er fem statuser totalt, så det vil gi false hvis statusen ikke er OPPRETTET eller UTREDES ?